.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           command: echo "make -C upstream-check/ -j2 verify && killall gpg-agent" | HOMEWORLD_CHROOT="$HOME/autobuild-chroot" USER="circleci" ./enter-chroot-ci.sh
       - run:
           name: Launch build with glass
-          command: echo "glass components -b root/master" | HOMEWORLD_CHROOT="$HOME/autobuild-chroot" USER="circleci" ./enter-chroot-ci.sh
+          command: echo "glass components -b root/master --branches-config /h/apt-branch-config/branches-ci.yaml" | HOMEWORLD_CHROOT="$HOME/autobuild-chroot" USER="circleci" ./enter-chroot-ci.sh
```

Comment:
Why not just copy branches-ci.yaml file over the standard location? The point is to test something close to what users will actually do.

Reply:
That's true, there isn't really a need to use a different branches config path.